### DISCUSSION TITLE
Understanding what GH pages is

### DISCUSSION THREAD

**Discussion by SHADOWELITE7** [community/community#21905] [Other Feature Feedback, Questions, & Ideas]
<p>So I have seen the video about GH pages and was wondering like when making a website do you create more pages on the repo or create another repo to make another page for your website</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/shadowelite7">@shadowelite7</a>,</p>
<p>If I understand you correctly, you’re asking about how to add multiple pages in a GitHub Pages site, correct? If so, you can absolutely have more than one page in your Pages site within the same repository. You can do this by creating additional Markdown files.</p>
<p>Does this help?</p>
<p>Thanks!</p>
